src/ResourceManager/Network/Commands.Network/help/New-AzureRmVirtualNetworkGatewayConnection.md

```diff
@@ -9,7 +9,7 @@ schema: 2.0.0
 # New-AzureRmVirtualNetworkGatewayConnection
 
 ## SYNOPSIS
-
+creates the Site-to-Site VPN connection between the virtual network gateway and the on-prem VPN device.
```

Comment:
Can you capitalize this line and the one below?

Reply:
Done